ledger/participant-integration-api/src/main/scala/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -80,6 +82,9 @@ final class StandaloneApiServer(
       authorizer = new Authorizer(Clock.systemUTC.instant _, ledgerId, participantId)
       healthChecksWithIndexService = healthChecks + ("index" -> indexService)
       executionSequencerFactory <- new ExecutionSequencerFactoryOwner()
+      servicesExecutionContext <- ResourceOwner
+        .forExecutorService(() => Executors.newWorkStealingPool())
```

Comment:
If we are to replace the direct execution context, a work-stealing thread pool is probably the best choice. Well done!

Reply:
I am a fan. :slightly_smiling_face:

Reply:
Well, what I meant is that the work-stealing pool seems well suited for the workload here, with many tasks which are likely small.